screens/Contacts/ContactListScreen.tsx

```diff
@@ -242,13 +226,8 @@ export const ContactListScreen = () => {
         />
       </View>
 
-      <View style={styles.container}>
-        <FlatList
-          data={filteredContacts}
-          renderItem={RenderOneContact}
-          keyExtractor={(contact) => contact.uid}
-        />
-      </View>
+      {selectedTab === "Plain View" ? RenderContactList() : RenderContactGraph()}
```

Comment:
If a graph computation will be a resourceful computation, we may have to cache the graph view so that it is not re-rendered on each switch between two sections. It would mean that both these components will have to become [top tabs](https://reactnavigation.org/docs/material-top-tab-navigator). Let's keep it like that for now though:)